serverMain/main.cpp

```diff
@@ -52,34 +63,52 @@ int main(int argc, char* argv[]) {
         exit(-1);
     }
     log.setOutputDest(util::Log::SPEEDLOG);
-    log.flush();
+    // FUTURE: add command line argument to turn `Log::_alwaysFlush` off.
+    log.setAlwaysFlush(true); // spdlog is highly prone to waiting a long time before writing to disk.
 
-    // Start the control system
-    control::Context::setup();
+    // Setup a simple signal handler to handle when clients closing connection results in SIGPIPE.
+    signal(SIGPIPE, signalHandler);
 
     // Start the telemetry server
+    LINFO("Starting Telemetry Server");
+    // FUTURE: get the correct entries into `system::Config`.
+    int const telemPort = 50001;
```

Comment:
In the future, maybe this `50001` can be assigned from the command line or configuration file. Same comment for the command server.